pkg/checks/traceroute/traceroute.go

```diff
@@ -0,0 +1,199 @@
+package traceroute
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/aeden/traceroute"
+	"github.com/caas-team/sparrow/internal/logger"
+	"github.com/caas-team/sparrow/pkg/checks"
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var _ checks.Check = (*Traceroute)(nil)
+
+const CheckName = "traceroute"
+
+type Target struct {
+	Addr string `json:"addr" yaml:"addr" mapstructure:"addr"`
+	Port uint16 `json:"port" yaml:"port" mapstructure:"port"`
+}
+
+// Config is the configuration for the traceroute check
+func NewCheck() checks.Check {
```

Comment:
I think you forgot to remove the godoc here. Please write a godoc for the constructor.